langgraph_prototype/test_langgraph_integration.py

```
@@ -0,0 +1,265 @@
+                print(f"最終応答プレビュー: {final_response[:100]}...")
+                
+                if intermediate_steps:
+                    print(f"実行ステップ: {len(intermediate_steps)}ステップ")
+                    for step in intermediate_steps:
+                        print(f"  - {step}")
+                
+            except Exception as e:
+                print(f"❌ テスト実行エラー: {e}")
+        
+        # 結果サマリー
+        print(f"\n📊 ルーティング精度テスト結果")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)